packages/components/src/internal/renderers/DefaultRenderer.tsx

```diff
@@ -55,8 +57,9 @@ export const DefaultRenderer: FC<Props> = memo(({ col, data }) => {
             }
 
             if (data.get('url')) {
+                const targetBlank = data.get('urlTarget') === TARGET_BLANK;
```

Comment:
I'm not sure why a user would do this, but if they use one of the other HTML <a> tag target attribute options, we would ignore it (i.e. "_top", "_self", etc.). I assume we are OK with that?

Reply:
This is for a client who wants to customize picklist grid to add a column that links to an external site.